shop/src/test/java/ro/msg/learning/shop/integration/TestBase.java

```diff
@@ -0,0 +1,18 @@
+package ro.msg.learning.shop.integration;
+
+import ro.msg.learning.shop.entity.*;
+
+public class TestBase {
```

Comment:
You have the same TestBase class (with the same data) also in the unit package. Please keep just one class in a common package which will be used by both integration and unit tests.